lib/elements/dom-repeat.html

```diff
@@ -250,12 +250,16 @@
         },
 
         /**
-         * When `initialCount` is used, this property defines a frame rate to
-         * target by throttling the number of instances rendered each frame to
-         * not exceed the budget for the target frame rate.  Setting this to a
+         * When `initialCount` is used, this property defines a frame rate (in
+         * fps) to target by throttling the number of instances rendered each
+         * frame to not exceed the budget for the target frame rate.  The
+         * framerate is effectively the number of `requestAnimationFrame`s that
+         * it tries to allow to actually fire in a given second, and it targets
```

Comment:
This sentence is kind of long. Maybe break sentence here: "... a given second. It does this by measuring the time between `rAF`s and continuously adjusting the number of items created each `rAF` to maintain the target framerate."

I also tried an optional rewording here to emphasize the fact that the process is ongoing rather than one-time. Totally optional if you prefer the current text.